cmd/juju/action/call.go

```diff
@@ -293,11 +313,69 @@ func (c *callCommand) Run(ctx *cmd.Context) error {
 			return err
 		}
 		d := FormatActionResult(result)
-		d["id"] = tag.Id()       // Action ID is required in case we timed out.
-		d["unit"] = unitTag.Id() // Formatted unit is nice to have.
-		info[result.Action.Receiver] = d
+		d["id"] = tag.Id() // Action ID is required in case we timed out.
+		info[unitTag.Id()] = d
+	}
+
+	if c.out.Name() == "plain" {
```

Comment:
That's interesting... Doesn't the registration of plain formatter on l.110 take care of this? 
I would have expected that only ```c.out.Write(..)``` is needed here without if-.

Reply:
ah yes, left over from a previous iteration, thanks